gossip/gossip_core.py

```diff
@@ -145,7 +145,6 @@ def __init__(self, node, **kwargs):
 
         self.PendingAckMap = {}
         self.MessageHandledMap = {}
-        self.MessageHandlerMap = {}
```

Comment:
MessageHandlerMap and MessageHandledMap are two different dicts. The MessageHandlerMap is a map of message types to handler functions and the MessageHandledMap is a map of handled messages where keys are message identifiers and values are message expiration times. Checkout the code comments from lines 85-118 in gossip_core.py. Thanks for looking through the code.